plugins/scaffolder-backend/src/scaffolder/stages/templater/helpers.ts

```diff
@@ -133,7 +133,7 @@ export const runDockerContainer = async ({
       // root on the host system on non Mac systems, because of reasons. Mainly the fact that
       // volume sharing is done using NFS on Mac and actual mounts in Linux world.
       // So we set the user in the container as the same user and group id as the host.
-      User: `${process.getuid()}:${process.getgid()}`,
+      User: `${process.getuid ? process.getuid() : ''}:${process.getgid ? process.getgid() : ''}`,
```

Comment:
I'm no expert but shouldn't it rather be that this field isn't given AT ALL if `process.getuid` and `.getgid` aren't defined? Doesn't sound sensible to specify `':'` here on win

Reply:
Agreed, I think we can just skip the `User` arg if on windows I think. And it should still work.